signing_service/signing_service/utils.py

```diff
@@ -0,0 +1,14 @@
+from golem_messages.cryptography import verify_pubkey
+from golem_messages.exceptions import InvalidKeys
+
+
+def is_valid_public_key(key):
```

Comment:
Please do some unit test only to this function.

Reply:
:+1: done